client/src/assets/icons/Loading.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable react/no-unknown-property */
```

Comment:
What is the reason for this eslint-disable?

Reply:
Solved. (Changed svg-properties)